index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Porfolio</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+        <header>
+
+            <nav>
+        
+                <ul class="navbar">
+                    <li>
+                    <a href="#"> <img src="images/logo.svg" alt="logo"> </a>
+                    </li>
+                    <li>
+                    <a href="#"> <img src="images/menu.svg" alt="menu"> </a>
+                    </li>
+
+                </ul>
+            </nav>
+
+        </header>
+
+        <section>
+            <div class="container">
+                <div class="text">
+                    <h1>I’m Mariane Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                <div class="connect">
+                    <p> LET'S CONNECT</p>
+                    <div class="logo"> 
+                        <a href="#"><img src="images/twitter.svg"> </a>
+                        <a href="#"><img src="images/linkedin.svg"> </a>
+                        <a href="#"><img src="images/medium.svg"> </a>
+                        <a href="#"><img src="images/github.svg"> </a>
+                        <a href="#"><img src="images/peace.svg"> </a>
+                    </div>
```

Comment:
Please make sure your social links follow correct sematics :point_down: 
The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href
for example instead of wrapping it in a `div `use a `<ul> t`ag and put each link in a `<li> `tag you have alredy placed them inside `<a>` tag which is correct :+1: .